src/index.html

```diff
@@ -4,9 +4,306 @@
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__links">
+        <a href="#homePage" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav>
+          <ul class="header__list">
+            <li class="header__link">
+              <a href="#Apple" class="header__link-a is-active">Apple</a>
+            </li>
+            <li class="header__link">
+              <a href="#Samsung" class="header__link-a">Samsung</a>
+            </li>
+            <li class="header__link">
+              <a href="#phones" class="header__link-a">Смартфоны и телефоны</a>
+            </li>
+            <li class="header__link" data-qa="nav-hover">
+              <a href="#laptops" class="header__link-a">Ноутбуки и компьютеры</a>
+            </li>
+            <li class="header__link">
+              <a href="#gadgets" class="header__link-a">Гаджеты</a>
+            </li>
+            <li class="header__link">
+              <a href="#tablet" class="header__link-a">Планшеты</a>
+            </li>
+            <li class="header__link">
+              <a href="#photo" class="header__link-a">Фото</a>
+            </li>
+            <li class="header__link">
+              <a href="#video" class="header__link-a">Видео</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main-content">
+      <section class="catalog main-content__catalog">
+        <section class="card catalog__card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          >
+          <div class="card__caption">
+            Моноблок APPLE A1419 iMac 27'' Retina 5K (MNED2UA/A)
+          </div>
+          <div class="card__code">
+            Код товара: 195434
+          </div>
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <div class="card__response">
+              Отзывов: 5
+            </div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Цена:</div>
+            <div class="card__price-numbers">69 999 грн</div>
+          </div>
+          <div class="button-buy" data-qa="card-hover">
+            <a href="" class="button-buy__link">КУПИТЬ</a>
+          </div>
```

Comment:
There is no real need to create separate class for this button, just use `a` tag and make it element of `card` block.
Also, this way the whole button is going to be clickable.